src/patternfly/components/DataList/data-list.scss

```diff
@@ -238,6 +245,65 @@
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
   }
 
+  // stylelint-disable
+  &.pf-m-draggable {
+    position: relative;
+    cursor: pointer;
+
```

Comment:
If a user should be able to click anywhere in the item row to drag the row (and clicking on any links/buttons/checks/etc in the row would do nothing), then I wonder if we should draw an `::after` pseudo element here with a `z-index` so that it overlays the row and makes clicking anywhere on the row trigger a click on the pseudo element.